culture/code-of-conduct.md

```diff
@@ -0,0 +1,129 @@
+# Code of Conduct
+
+```{toctree}
+:hidden:
+response-protocol
+```
+
+## Why have a code of conduct?
+
+Our goal is to create the best community in the world for learning, using, and creating open infrastructure for interactive computing.
+We want every member of the 2i2c community to be able to focus their full attention on building amazing tools and developing a thriving culture of collaboration and inclusion around them.
+This is impossible to do if you are being harassed, stalked, or discriminated against.
+
+Accordingly, anyone that participates in a 2i2c space is expected to show respect and courtesy to each other in all interactions, whether in GitHub repositories, our Slack channel, in in-person events, when representing 2i2c in public, or in events and spaces associated with {term}`ICSI`.
+
+To make sure that everyone has a common understanding of “show respect and courtesy to each other,” we have adopted the following code of conduct. The code of conduct is enforced by the 2i2c Executive Director and the Steering Council.
+
+## Unacceptable behavior
+
+The following types of behavior are unacceptable in 2i2c spaces, both online and in-person, and constitute code of conduct violations.
+
+### Abusive behavior
+
+Harassment
+: including offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, or religion, as well as sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, inappropriate physical contact, and unwelcome sexual or romantic attention.
+
+Threats
+: threatening someone physically or verbally. For example, threatening to publicize sensitive information about someone’s personal life.
+
+### Unwelcoming behavior
+
+
+Blatant -isms
+: saying things that are explicitly racist, sexist, homophobic, etc. For example, arguing that some people are less intelligent because of their gender, race or religion. {ref}`Subtle -isms <social:subtle-isms>` and small mistakes made in conversation are not code of conduct violations. However, repeating something after it has been pointed out to you that you broke a social rule, or antagonizing or arguing with someone who has pointed out your subtle -ism is considered unwelcoming behavior, and is not allowed at 2i2c.
+
+Maliciousness towards other community members
+: deliberately attempting to make others feel bad, name-calling, singling out others for derision or exclusion. For example, telling someone they’re not a real programmer or that they don’t belong at 2i2c.
+
+Being especially unpleasant
+: for example, if we’ve received reports from multiple 2i2c users, team members, or collaborators of agitating, rude, or especially distracting behavior over an extended period of time.
```

Comment:
Again, this feels vague. "Especially" implies a comparison to some baseline. What is the baseline?